### ISSUE TITLE
Can you tell me how this code test?

### ISSUE THREAD

**Issue by yancydong** [pnborchert/MultiRep#1]

**Comment by pnborchert** [comment]
Hi, you first need to download the dataset from this repository: https://github.com/thunlp/FewRel. Feel free to post your error messages here.

**Comment by yancydong** [comment]
When I was running the code, I found that only the training set and validation set were used, and I didn't know how to output the results of the test set.

**Comment by yancydong** [comment]
![image](https://github.com/pnborchert/MultiRep/assets/123464994/c8b5fbd7-25e4-4e3e-a156-c611b29b3bf5)
It only use the training set and validation set.

**Comment by PPPPPPPs** [comment]
The test set is here
https://codalab.lisn.upsaclay.fr/competitions/7395#participate-get-data

**Comment by XxxAtlantis** [comment]
If I want to use my own data for practical applications with this model, how should I organize the dataset?
